provider/gke/gke.go

```diff
@@ -300,6 +304,7 @@ func (c *GKE) NodePoolDelete(*kingpin.ParseContext) error {
 	return nil
 }
 
+// nodePoolDeleted checks whether a nodepool has been deleted or not
```

Comment:
missing dots in many places.

Reply:
// nodePoolDeleted checks whether a nodepool has been deleted.
just like this is fine.